blackbox/docs/config/logging.rst

```diff
@@ -99,3 +99,42 @@ Settings<sys-cluster-settings>`).
 
 .. _Log4j: https://logging.apache.org/log4j/1.2/
 .. _`hierarchical named loggers`: https://logging.apache.org/log4j/1.2/manual.html
+
+.. _conf-logging-gc:
+
+Garbage Collection Logging
+==========================
+
+Addition to the regular application logging, CrateDB also logs garbage
+collection times of the Java Virtual Machine using the built-in GC logging of
+the JVM.
+
+GC logging is enabled by default since CrateDB 3.0. It can be disabled by
+setting the environment variable ``CRATE_DISABLE_GC_LOGGING``.
+
+The default location for the GC log files is ``$CRATE_HOME/logs``. The files
+are logrotated at 64MB in size, kept 16 rounds, and prefixed with ``gc.log``.
```

Comment:
replace with "rotated" and maybe link to the docs of logrotate or somethin?

also "kept 16 rounds" needs clarification

Reply:
could you say: are kept for 16 rotations? - does this make sense?